### PR TITLE
Add soroban metrics for selected network limits

### PR DESCRIPTION
**Pull request by jayz22**
# Description

Add some network limits to soroban Medida metrics, so that they can be displayed together with the other Soroban metrics and put them into context. 

# Checklist
- [ ] Reviewed the [contributing](https://github.com/stellar/stellar-core/blob/master/CONTRIBUTING.md#submitting-changes) document
- [ ] Rebased on top of master (no merge commits)
- [ ] Ran `clang-format` v8.0.0 (via `make format` or the Visual Studio extension)
- [ ] Compiles
- [ ] Ran all tests
- [ ] If change impacts performance, include supporting evidence per the [performance document](https://github.com/stellar/stellar-core/blob/master/performance-eval/performance-eval.md)

### CODE REVIEW

File: src/transactions/InvokeHostFunctionOpFrame.cpp [hunk 1]
```diff
@@ -346,6 +407,7 @@ InvokeHostFunctionOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx,
     HostFunctionMetrics metrics(app.getMetrics());
     auto const& sorobanConfig =
         app.getLedgerManager().getSorobanNetworkConfig();
+    metrics.noteLimits(sorobanConfig);
```
Comment (dmkozh) [inline src/transactions/InvokeHostFunctionOpFrame.cpp:410]: This is not the proper place for monitoring this. We shouldn't emit config more than once per ledger. On the other hand, if a ledger has 0 invoke host fn ops, we would skip this. We should monitor this somewhere during the ledger close e.g. in the ledger manager.

Comment (jayz22) [reply src/transactions/InvokeHostFunctionOpFrame.cpp]: Makes sense, I moved it to `closeLedger` in the ledger manager.

File: src/ledger/LedgerManagerImpl.cpp [hunk 2]
```diff
@@ -1201,6 +1279,11 @@ LedgerManagerImpl::advanceLedgerPointers(LedgerHeader const& header,
     mLastClosedLedger.header = header;
 }
 
+void
```
Comment (dmkozh) [inline src/ledger/LedgerManagerImpl.cpp:1282]: This is not used, please remove.

Comment (jayz22) [reply src/ledger/LedgerManagerImpl.cpp]: Ah sorry, removed

**Comment by dmkozh** [comment]
r+ 651e9f3

**Pull request merged**